src/fhir/PlanDefinition_cove.json

```diff
@@ -0,0 +1,438 @@
+      "id": "Day162Msg0",
+      "name": "Day 162 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 162,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "13:39:35"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "I have a quote for you: \"The greatest glory in living lies not in never falling, but in rising every time we fall.\" (Nelson Mandela) Hope you are finding ways to keep getting back up! 🌱"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day220Msg0",
+      "name": "Day 220 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 220,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "14:04:58"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Hey {name}, I know some days can be hard but we are cheering for you! 🎉"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day265Msg0",
+      "name": "Day 265 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 265,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "15:57:38"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Hi {name}, we hope this message finds you in a good place.  Wherever you are today, you are valued, and you are enough. 🙌 If you’d like to chat about how things are going, I’d be happy to hear from you. -Riley"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day317Msg0",
+      "name": "Day 317 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 317,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "14:37:59"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Thinking of you, {name} 🙂 I'm not always available but if you need to talk to someone, the National Hotline number is 988 & press 1 and the Veterans Crisis Line number is 838-255. -Riley"
```

Comment:
@mcjustin Here is another instance `Riley`, which I am not sure if meant for it be author name, i.e. {userName}

Reply:
per above